tests/data/permutations/ru_permutations_tools4noobs.csv

```diff
@@ -0,0 +1,24 @@
+number,text

```

Comment:
This looks to be the same as `` tests/data/permutations/ru_permutations_mathtools.csv``, is this expected?